### PR TITLE
[Wait for #1882] [ Interpreter ] add batch normalziation realizer @open sesame 04/26 10:37

### PR DESCRIPTION
**Pull request by jijoongmoon**
This patch removes the batch normalization layers with bn_realizer in
the GraphRepresentation for TfliteInterperter which is for the
inference.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

**Review by songgot — APPROVED** [review]
Looks good to me

File: nntrainer/compiler/bn_realizer.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#include <stdexcept>
+#include <unordered_map>
+
+namespace nntrainer {
+
+static constexpr size_t SINGLE_INOUT_IDX = 0;
+
+BnRealizer::BnRealizer() {}
+
+BnRealizer::~BnRealizer() {}
```
Comment (mhs4670go) [inline nntrainer/compiler/bn_realizer.cpp:29]: Just curiosity, is there any reason why these default constructor and destructor are declared even though they are implicitly done?

Comment (jijoongmoon) [reply nntrainer/compiler/bn_realizer.cpp:29]: No reason. It's just from the skeleton code. I will remove them in a later pr. Thanks.

Comment (mhs4670go) [reply nntrainer/compiler/bn_realizer.cpp:29]: If it is just convention things, don't bother removing them:) Anyway looks good to me.

**Review by lhs8928 — APPROVED** [review]
LGTM

**Pull request merged**